index.html

```diff
@@ -576,42 +607,26 @@ <h3>The <dfn data-lt="pointerup"><code>pointerup</code> event</dfn></h3>
-                    <li>As part of the drag operation initiation algorithm as defined in the <a data-cite="html/#drag-and-drop-processing-model">drag and drop processing model</a> [[HTML]],
-                     for the pointer that caused the drag operation.</li>
-                </ul>
-                <p>After firing the <code>pointercancel</code> event, the user agent MUST also fire a pointer event named <code>pointerout</code> followed by firing a pointer event named <code>pointerleave</code>, and <a href="#implicit-release-of-pointer-capture">implicitly release the pointer capture</a> if the pointer is currently captured.</p>
-                <div class="note">
-                    <p><em>This section is non-normative.</em></p>
-                    <p>Examples of scenarios in which the user agent might determine that a pointer is unlikely to continue to produce events include:
-                    <ul>
-                        <li>A device's screen orientation is changed while a pointer is active.</li>
-                        <li>The user inputs a greater number of simultaneous pointers than is supported by the device.</li>
-                        <li>The user agent interprets the input as accidental (for example, the hardware supports palm rejection).</li>
-                        <li>The user agent interprets the input as a pan or zoom gesture.</li>
-                    </ul>
-                    <p>Methods for changing the device's screen orientation, recognizing accidental input, or using a pointer to manipulate the viewport (e.g. panning or zooming) are out of scope for this specification.</p>
-                </div>
+                <p>The user agent MUST fire a pointer event named <code>pointercancel</code> when it detects a scenario to <a>suppress a pointer event stream</a>.</p>
             </section>
             <section>
                 <h3>The <dfn data-lt="pointerout"><code>pointerout</code> event</dfn></h3>
                 <p>The user agent MUST <a>fire a pointer event</a> named <code>pointerout</code> when any of the following occurs:</p>
                 <ul>
-                    <li>A pointing device is moved out of the hit test boundaries of an element. Note that <code>setPointerCapture()</code> or <code>releasePointerCapture()</code> might have changed the hit test target and while a pointer is captured it is considered to be always inside the boundaries of the capturing element for the purpose of firing boundary events.</li>
+                    <li>The pointing device is moved out of the hit test boundaries of an element.  Note that <code>setPointerCapture()</code> or <code>releasePointerCapture()</code> might have changed the hit test target and while a pointer is captured it is considered to be always inside the boundaries of the capturing element for the purpose of firing boundary events.</li>
```

Comment:
`hit test target and while a pointer` maybe add a full stop to break up the overly long sentence a bit `hit test target. While a pointer`